menpowidgets/base.py

```diff
@@ -650,7 +650,7 @@ def update_widgets(change):
 
 
 def visualize_images(images, figure_size=(10, 8), style='coloured',
-                     browser_style='buttons'):
+                     browser_style='buttons', custom_function=None):
```

Comment:
I think that the `custom_info_callback` variable name initially proposed by @patricksnape is more clear than `custom_function`.